rs/nns/governance/feature_flags.bzl

```
@@ -5,6 +5,6 @@ Feature flags for turning TLA-based checks on and off.
 def test_with_tla():
     return select({
         # Remove "tla" here to disable TLA-related checks in the CI
```

Could you explain why you're disabling TLA here? (I know it's failing the pipeline, but is there nothing better that can be done in this case?)

```suggestion
        # Add "tla" here to enable TLA-related checks in the CI
```